lib/slide_hub/convert_procedure.rb

```diff
@@ -105,7 +105,7 @@ def upload_files
 
     def update_database
       slide = Slide.where('slides.object_key = ?', object_key).first
-      slide.convert_status = 100
+      slide.converted!
```

Comment:
__[Reek]__

[FeatureEnvy] ConvertProcedure#update_database refers to 'slide' more than self (maybe move it to another class?)


<!-- Comment by [SideCI](https://sideci.com) -->